src/other/Table.js

```diff
@@ -23,11 +23,12 @@
 
     Table.prototype.publish("renderHtmlDataCells", false, "boolean", "enable or disable HTML within cells",null,{tags:["Private"]});
     Table.prototype.publish("pagination", false, "boolean", "Enable or disable pagination",null,{tags:["Private"]});
+    Table.prototype.publish("paginationLimit", null, "number", "Maximum number of rows allowed before pagination defaults to on",null,{tags:["Private"]});
```

Comment:
@jbrundage this is in response to https://github.com/hpcc-systems/Visualization/issues/1140.  Based on your discussion with Jo, should this default to a certain number, or leave it up to the dev?